Packs/RunZero/Integrations/RunZero/RunZero.py

```
@@ -0,0 +1,525 @@
+    message['Explorer'] = raw.get('agent_name', '')
+    message['Hosted_Zone'] = raw.get('hosted_zone_name', '')
+    message['Site'] = raw.get('site_name', '')
+    return [message]
+
+
+def parse_raw_quota_get(raw: dict) -> dict:
+    message = {}
+    message['usage_limit'] = raw.get('created_at', '')
+    message['usage_today'] = raw.get('usage_today', '')
+    message['counter'] = raw.get('counter', '')
+    return message
+
+
+def parse_raw_wireless(raw: dict) -> list:
+    message = {}
+    message['ID'] = raw.get('id', '')
+    message['ESSID'] = raw.get('essid', '')
+    message['BSSID'] = raw.get('bssid', '')
+    message['Vendor'] = raw.get('vendor', '')
+    message['Family'] = raw.get('family', '')
+    message['Type'] = raw.get('type', '')
+    message['Auth'] = raw.get('authentication', '')
+    message['Enc'] = raw.get('encryption', '')
+    message['Sig'] = raw.get('signal', 0)
+    message['Int'] = raw.get('interface', '')
+    message['Additional'] = raw.get('data', {})
+    message['First_seen'] = timestamp_to_datestring(raw.get('created_at', ''))
+    message['Last_seen'] = timestamp_to_datestring(raw.get('last_seen', ''))
+    message['Site'] = raw.get('site_name', '')
+    return [message]
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def asset_search(client: Client, args: dict) -> CommandResults:
+    search_string = ''
+    limit = arg_to_number(args.get('limit', DEFAULT_LIMIT))
+    if args.get('ips'):
+        search_string = ','.join(argToList(args.get('ips')))
+        search_string = f'?search=address:{search_string}'
+    elif args.get('hostnames'):
+        search_string = ','.join(argToList(args.get('hostnames')))
+        search_string = f'?search=name:{search_string}'
+    elif args.get('asset_id'):
+        search_string = ','.join(argToList(args.get('asset_id')))
+        search_string = f'/{search_string}'
+    elif args.get('search'):
+        search_string = f'?search={args.get("search")}'
```

Since using several types simultaneously is incompatible - please raise an error if more than one is provided.